speckle/base/client.py

```diff
@@ -161,8 +138,8 @@ def login_with_token(self, token):
             'Authorization': token,
         })
 
-        if self.me is None:
-            self.me = {}
+        full_profile = self.accounts.get_profile()
```

Comment:
Where is `self.accounts`? It doesn't seem to be a member of `SpeckleApiClient`...

Reply:
I think it is inherited from a resource or something? I found it from this code sample in `acounts.py`

```
                >>> from speckle.SpeckleClient import SpeckleApiClient
                >>> client = SpeckleApiClient(host='hestia.speckle.com')
                >>> client.login(email='test@test.com', password='somesupersecret')
                >>> client.accounts.get_profile()
```
I did test it and it works! :)

Reply:
Ah ok, found it :) Looks like it requests the user details from the server, based on the authorization token. Need to check how it differs from the `me` response from `login()`.